File: CS/DependentEditors/XtraReport1.cs
using DevExpress.XtraReports.UI;
// ...

namespace DependentEditors {
    public partial class XtraReport1 : XtraReportBase {
        public XtraReport1() {
            InitializeComponent();

            startDate.DataBindings.Add(new XRBinding(Parameters[startDateParameterName], 
                "Text", "{0:dd.MM.yyyy}"));
            endDate.DataBindings.Add(new XRBinding(Parameters[endDateParameterName], 
                "Text", "{0:dd.MM.yyyy}"));
            period.DataBindings.Add(new XRBinding(Parameters[periodParameterName], 
                "Text", string.Empty));
        }
    }
}

File: VB/DependentEditors/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraReports.UI
' ...

Namespace DependentEditors
	Partial Public Class XtraReport1
		Inherits XtraReportBase
		Public Sub New()
			InitializeComponent()

			startDate.DataBindings.Add(New XRBinding(Parameters(startDateParameterName), "Text", "{0:dd.MM.yyyy}"))
			endDate.DataBindings.Add(New XRBinding(Parameters(endDateParameterName), "Text", "{0:dd.MM.yyyy}"))
			period.DataBindings.Add(New XRBinding(Parameters(periodParameterName), "Text", String.Empty))
		End Sub
	End Class
End Namespace